general/general-contributing.asciidoc

```diff
@@ -1,15 +1,11 @@
 = Contributing Guide
 
-include::Contributing-Code.asciidoc[leveloffset=1]
+include::../.github/CONTRIBUTING.asciidoc[leveloffset=1]
 
-include::Contributing-Code-of-Conduct.asciidoc[leveloffset=1]
-
-include::Contributing-Development-Guidelines.asciidoc[leveloffset=1]
+include::../.github/CODE_OF_CONDUCT.md[leveloffset=1]
 
 include::Contributing-Git-Fork-Guide.asciidoc[leveloffset=1]
```

Comment:
Check if it is possible to put this file inside .github repository so we have related topics in the same folder, then you could simply create a file like:

// master-github.asciidoc
= Contributing guide

include::../.github/CONTRIBUTING.asciidoc[leveloffset=1]
include::../.github/CODE_OF_CONDUCT.md[leveloffset=1]
include::Contributing-Git-Fork-Guide.asciidoc[leveloffset=1]

// in devonfw-guide's master.asciidoc
// IMPORTANT: this just before release notes
include:github/master-github.asciidoc